### PR TITLE
CAMEL-13617: Improve testability of google-sheets component

### PR DESCRIPTION
**Pull request by christophd**

### CODE REVIEW

File: components/camel-google-sheets/pom.xml [hunk 1]
```diff
@@ -40,8 +40,55 @@
     <camel.osgi.private.pkg>org.apache.camel.component.google.sheets.internal</camel.osgi.private.pkg>
     <camel.osgi.export.pkg>org.apache.camel.component.google.sheets</camel.osgi.export.pkg>
     <camel.osgi.export.service>org.apache.camel.spi.ComponentResolver;component=google-sheets</camel.osgi.export.service>
+
+    <citrus.version>2.8.0</citrus.version>
```
Comment (oscerd) [inline components/camel-google-sheets/pom.xml]: maybe these placeholders can be in the parent/pom.xml, for the future we can use them directly and have a common version in all components

Comment (christophd) [reply components/camel-google-sheets/pom.xml]: very true! can I force push the change to this PR?

Comment (oscerd) [reply components/camel-google-sheets/pom.xml]: Sure!

Comment (christophd) [reply components/camel-google-sheets/pom.xml]: Done!

File: components/camel-google-sheets/src/test/resources/test-options.properties [hunk 2]
```diff
@@ -19,8 +19,8 @@
 ## Login properties for Google Sheets Component
 #####################################
 ## Application client id and secret
-clientId=
-clientSecret=
+clientId=syndesis-client
```
Comment (oscerd) [inline components/camel-google-sheets/src/test/resources/test-options.properties]: Are this related to a different project?

**Comment by oscerd** [comment]
Ok, I fixed CS for this one too and merged on 2.x, thanks
